projects/kernel-config/Makefile

```diff
@@ -12,7 +12,7 @@
 HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 # Name on Hub
 IMAGE:=kernel
-ARCH?=$(shell if [ "$$(arch)" = "x86_64" ]]; then echo x86; else $$(arch); fi)
+ARCH?=x86
```

Comment:
Good question. I think this is probably leftover from some other idea I had when I was playing around with this and munging defconfig in the kernel tree. I don't see any reason right now why we can't switch it, so I'll add a patch to this PR to do that.